CONTRIBUTING.md

```diff
@@ -65,41 +65,59 @@ Steps to get started:
 
 ## Getting your development environment set up
 
-After cloning the project there are a few steps required to get the project running.
+After cloning the project there are a few steps required to get the project running. 
+For running all tests, take the following steps:
 
-1.  Fetch submodules to pull ECMAScript test suites.
+1. Fetch submodules to pull ECMAScript test suites.
 
     ```bash
     git submodule update --init --recursive
     ```
 
-2.  Install js dependencies.
+2. Install js dependencies.
 
     ```bash
     yarn
     ( cd crates/swc_ecma_transforms; yarn )
     ```
 
-3.  Setup some environment variables which is required for tests.
+3. Setup some environment variables which is required for tests.
 
     ```bash
-    export RUST_BACKTRACE=full
+    export RUST_BACKTRACE=0
```

Comment:
This seems wrong?

Reply:
see my description, perhaps theres something else that needs to be configured?

Reply:
We need `RUST_LIB_BACKTRACE=0`

Reply:
We can configure it in the error reporting test, and I think we should.

Reply:
@kdy1 Great catch on this. `RUST_BACKTRACE=0` Was a suggestion I dug up on the discord server.

The error output is deceiving: it looked like it was emitting a backtrace... but actually, the captured error was being passed into `NormalizedOutput` with `Debug` formatting. 

With backtrace enabled, the lint errors were being captured with stack traces enabled. This is an easy fix: https://github.com/swc-project/swc/pull/3298/files#diff-d807373603a279f7b50a670e39c4b65357c79d13622bca50583de75692c3b6baL76